### PR TITLE
Upgrade to Storefront API version 2019-10

### PR DESCRIPTION
**Pull request by rebeccajfriedman**
- Bumps the schema to include the following changes: https://help.shopify.com/en/api/versioning/release-notes/2019-10#storefront-api-changes
- Removes some references to the hardcoded api version

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -1338,7 +1338,7 @@
         {
           "kind": "SCALAR",
           "name": "DateTime",
-          "description": "An ISO-8601 encoded UTC date time string.",
+          "description": "An ISO-8601 encoded UTC date time string. Example value: `\"2019-07-03T20:47:55Z\"`.",
```
Comment (jmignac) [inline schema.json]: Maybe we could try to have current version as the example?

File: test/client-test.js [hunk 2]
```diff
@@ -27,7 +28,7 @@ suite('client-test', () => {
     new Client(new Config(config), FakeGraphQLJSClient); // eslint-disable-line no-new
 
     assert.deepEqual(passedTypeBundle, types);
-    assert.equal(passedUrl, 'https://sendmecats.myshopify.com/api/2019-07/graphql');
+    assert.equal(passedUrl, `https://sendmecats.myshopify.com/api/${config.apiVersion}/graphql`);
```
Comment (jmignac) [inline test/client-test.js]: 👍

File: schema.json (hunk 1 above)
Comment (rebeccajfriedman) [reply schema.json]: This is pulling from Storefront API documentation. You are welcome to change the description there, if you'd like.

File: test/client-integration-test.js [hunk 3]
```diff
@@ -23,16 +23,16 @@ import shopPoliciesFixture from '../fixtures/shop-policies-fixture';
 
 suite('client-integration-test', () => {
   const domain = 'client-integration-tests.myshopify.io';
-  const apiVersion = '2019-07';
-  const apiUrl = `https://${domain}/api/${apiVersion}/graphql`;
   const config = {
     storefrontAccessToken: 'abc123',
     domain
```
Comment (osamie) [inline test/client-integration-test.js]: Not directly related to your change but I couldn't help but notice this dangling reference. What is domain mapped to in this dictionary?

Comment (DrTeePot) [reply test/client-integration-test.js]: It's shorthand for `domain: domain` so it has a value of `'client-integration-tests.myshopify.io'`

Comment (osamie) [reply test/client-integration-test.js]: Thanks. Just looked it up - _object literal property value shorthand_ - apparently it's a new syntactic sugar introduced in es6

**Pull request merged**